mix.exs

```diff
@@ -49,7 +49,8 @@ defmodule Protein.Mixfile do
 
   defp deps do
     [
-      {:amqp, "~> 0.2"},
+      {:amqp, "~> 1.3.2"},
+      {:jsx, "~> 2.9", override: true},
```

Comment:
Is it safe to override version in library deps?

Reply:
In general not rly but in this case it seems to work just fine.